src/sonic-yang-models/yang-models/sonic-dscp-fc-map.yang

```diff
@@ -0,0 +1,68 @@
+module sonic-dscp-fc-map {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-dscp-fc-map";
+
+    prefix dtm;
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "DSCP_TO_FC_MAP yang Module for SONiC OS";
+
+    revision 2021-10-29 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-dscp-fc-map {
```

Comment:
I will reviews, thanks for reviewing the rest.

Reply:
@smaheshm since Praveen has approved the other changes is this PR ready to be merged in?
I've made a small update after his approval to correct some code comments only.